sass/_SortDropdown.scss

```diff
@@ -0,0 +1,23 @@
+@import 'mixins/resultsHeaderElement';
+@import 'Variables';
+
+.CoveoSortDropdown {
+  .coveo-dropdown {
+    @include resultsHeaderElement();
+
+    text-indent: 0;
+    margin: 0;
+    border: none;
+    cursor: pointer;
+    background-position-y: -5px;
+    font-family: 'Lato', $helvetica, sans-serif;
```

Comment:
Why do we need to specify the font-family here ? Is it not inherited from a more global rule that applies to the whole search interface ?

Reply:
For `select` elements `font-family` inheritance doesn't work automatically, but` font-family: inherit `will do the job, without no hard-coding the font-family.